invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides

```diff
@@ -14,6 +14,10 @@ body.cover-page{
   background-color: transparent;
 }
 
+.custom-message{
+  width: 93.3% !important;
```

Comment:
It's not possible to fix via using the SUI CSS classes? I would think that it was a problem with the grid somehow?

Reply:
Yes it is, we can just fill the container with the message and add some padding to it